src/cartridges/int_adyen_SFRA/cartridge/client/default/js/adyen_checkout/checkoutConfiguration.js

```diff
@@ -128,11 +130,17 @@ function getGiftCardConfig() {
         async: false,
         success: (data) => {
           giftcardBalance = data.balance;
+          document.querySelector('button[value="submit-payment"]').disabled = false;
           if (data.resultCode === constants.SUCCESS) {
-            const giftCardSelect = document.querySelector('#giftCardSelect');
+            const { giftCardsInfoMessageContainer, giftCardSelect } = getGiftCardElements();
             if (giftCardSelect) {
               giftCardSelect.classList.add('invisible');
             }
+            document.querySelector('button[value="submit-payment"]').disabled = true;
```

Comment:
In this case, if the user redeems a giftcard, and then he wants to pay with another payment method, the place order button will be blocked.

Reply:
the condition (data.resultCode === constants.SUCCESS) is true only when the gift card balance in enough to pay for complete order and so the gift card Pay button is shown.